services/blockchain-connector/shared/sdk/auth.js

```diff
@@ -31,6 +31,19 @@ const getAuthAccount = async (address) => {
 	}
 };
 
+const getAuthMultiSigRegMsgSchema = async () => {
+	try {
+		const multiSigRegMsgSchema = await invokeEndpoint('auth_getMultiSigRegMsgSchema');
+		return multiSigRegMsgSchema;
+	} catch (err) {
+		if (err.message.includes(timeoutMessage)) {
+			throw new TimeoutException('Request timed out when calling \'getMultiSigRegMsgSchema\'.');
+		}
+		throw err;
```

Comment:
We can add warn message to make it consistent
```suggestion
		if (err.message.includes(timeoutMessage)) {
			throw new TimeoutException('Request timed out when calling \'getMultiSigRegMsgSchema\'.');
		}
		logger.warn(`Error returned when invoking 'auth_getMultiSigRegMsgSchema'.\n${err.stack}`);
		throw err;
```

Reply:
Applied.